public/js/admin.js

```diff
@@ -441,7 +441,7 @@ var updateVisibility = function(qid) {
         },
         success: function(data) {
             displayQuestionTable();
-            dropSnack(colours.SUCCESS_GREEN, 'Question ' + qid + 'is now visible to the students');
+            dropSnack(colours.SUCCESS_GREEN, 'Question ' + qid + ' is now visible to the students');
```

Comment:
This is apart of PR #36

Reply:
Yes that a mistake, I will fix it once 36 is merged